server/util/clickhouse/schema/schema.go

```diff
@@ -157,6 +157,10 @@ type Execution struct {
 
 	Stage int64
 
+	TargetID        string
+	ActionMnemonic  string
+	ConfigurationID string
```

Comment:
Sorry, just thinking about this a bit more, I know this is also what the protocol provides, but we currently have this awkward split between linux + mac RBE minutes, and this string doesn't make it terribly easy to filter for the difference--it's specific to our cloud deployment, i guess, but maybe we could do something to distinguish these two since they bill differently?

Reply:
Discussed this with Jim on Slack and we concluded that the Mac vs Linux RBE minutes information should originate from BuildBuddy Executors, while these fields are populated from Bazel clients and thus, unrelated.